wake_word.py

```diff
@@ -62,12 +84,23 @@ def process(self, text):
                 ]
             )
             message_content = response.choices[0].message['content'].strip()
+
+            # Token calculation for response
+            response_tokens = self.count_tokens(message_content)
+            if response_tokens > 40:  # Ensure response tokens do not exceed 40
```

Comment:
Do you think we should not put a limit to response? OpenAI is going to charge us anyway, why not receive full response. Notice that you control the `temperature` of response while making the call to OpenAI

Reply:
we have a limit of around 100 tokens per minute so if we change the output response it will cost us more because the number of output tokens will increase .and tokens are dependant on the words you are using .special characters use more tokens then simple words.

Reply:
I understand but I think we are mixing "input tokens" (what we send to OpenAI) and "response tokens" (what OpenAI send us in response to our prompt). We shouldn't have any limit on response, or a very big limit on response, like 1000 or something.